storage/rocksdb/ha_rocksdb.cc

```diff
@@ -11831,12 +11817,18 @@ bool ha_rocksdb::skip_unique_check() const {
          any indexes. If the table has secondary keys, then those might becomes
          inconsisted/corrupted
       3) We're using read-free replication
+      4) For CREATE INDEX, DROP INDEX, & ALTER TABLE when any duplicate key
```

Comment:
For DDL, we don't need rocksdb lock but still need unique check.

Reply:
The use of writebatch needs more consideration: 1) it effectively forces 1PC protocol; 2) how do locks not being taken fare if i.e. concurrent writes are present?

I have a vague TODO item "use writebatch more", but I'd prefer to address it separately

Reply:
yes.. please address it separately...